themes/default/views/administrate/setup/interface_editor/screen_html.php

```diff
@@ -25,33 +25,35 @@
  *
  * ----------------------------------------------------------------------
  */
- 	$t_subject = $this->getVar('t_subject');
-	$vn_subject_id = $this->getVar('subject_id');
-	
-	$t_ui = $this->getVar('t_ui');
-	
-	print $vs_control_box = caFormControlBox(
-		caFormSubmitButton($this->request, __CA_NAV_BUTTON_SAVE__, _t("Save"), 'InterfaceEditorForm').' '.
-		caNavButton($this->request, __CA_NAV_BUTTON_CANCEL__, _t("Cancel"), '',  'administrate/setup/interface_editor', 'InterfaceEditor', 'Edit/'.$this->request->getActionExtra(), array('ui_id' => $vn_subject_id)), 
-		'', 
-		(intval($vn_subject_id) > 0) ? caNavButton($this->request, __CA_NAV_BUTTON_DELETE__, _t("Delete"), '', 'administrate/setup/interface_editor', 'InterfaceEditor', 'Delete/'.$this->request->getActionExtra(), array('ui_id' => $vn_subject_id)) : ''
-	);
+$t_subject = $this->getVar('t_subject');
+$vn_subject_id = $this->getVar('subject_id');
+
+$t_ui = $this->getVar('t_ui');
+
+print $vs_control_box = caFormControlBox(
+	caFormSubmitButton($this->request, __CA_NAV_BUTTON_SAVE__, _t("Save"), 'InterfaceEditorForm').' '.
+	caNavButton($this->request, __CA_NAV_BUTTON_CANCEL__, _t("Cancel"), '',  'administrate/setup/interface_editor', 'InterfaceEditor', 'Edit/'.$this->request->getActionExtra(), array('ui_id' => $vn_subject_id)),
+	'',
+	(intval($vn_subject_id) > 0) ? caNavButton($this->request, __CA_NAV_BUTTON_DELETE__, _t("Delete"), '', 'administrate/setup/interface_editor', 'InterfaceEditor', 'Delete/'.$this->request->getActionExtra(), array('ui_id' => $vn_subject_id)) : ''
+);
 ?>
-	<div class="sectionBox">
+<div class="sectionBox">
+<?php
+	print caFormTag($this->request, 'Save/'.$this->request->getActionExtra().'/ui_id/'.$vn_subject_id, 'InterfaceEditorForm', null, 'POST', 'multipart/form-data');
+?>
+		<div class="grid">
```

Comment:
This line `<div class="grid">` and its corresponding `</div>` (line 55 in this file) is the real change.